.github/workflows/PR.yml

```diff
@@ -0,0 +1,26 @@
+name: Label and Assign PRs
+on:
+  pull_request:
+    types:
+      - opened
+jobs:
+  label_and_assign_PRs:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - run: |
+          gh pr edit "$NUMBER" --add-label "$LABELS"
+          gh pr edit "$NUMBER" --add-assignee "$ASSIGNEE"
+          curl \
+            -X POST \
+            -H "Accept: application/vnd.github.v3+json" \
+            -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+            https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/requested_reviewers \
+            -d '{"team_reviewers": ["Web Development Team"]}'
+        env:
```

Comment:
You can define environment variables outside of the run step. I would do this to increase readability. Maybe put it between `on:` and `jobs:`